content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,97 @@
+
+Broad topics included [SPECs](https://scientific-python.org/specs/), documentation, tools & bots, [lectures](https://lectures.scientific-python.org/), `scipy.sparse`, telemetry, Array API, and type annotation.
+
+### Documentation
+
+Documentation was a much more popular topic than anticipated!
+
+- The new [mystmd](https://mystmd.org/guide) tooling generated some excitement, and an [experimental port of the NumPy tutorials](https://github.com/numpy/numpy-tutorials/tree/mystjs) was made.
+- Recommendations on consistent use of [backticks](https://github.com/numpy/numpydoc/pull/525) and [monospaced font](https://github.com/pydata/pydata-sphinx-theme/issues/1852) were submitted to numpydoc and pydata-sphinx-theme, respectively.
+
+### SPECs
+
+The Scientific Python Ecosystem Coordination documents (SPECs) aim to improve coordination of technical development across the ecosystem.
+
+Several new SPECs were started:
+
+- [SPEC-?: Dispatching (`spatch`)](https://hackmd.io/yI1iAqekQIq0a4jLS9WPyw)
+- [SPEC-8: Securing The Release Process](https://scientific-python.org/specs/spec-0008/)
+- [SPEC-9: Governance](https://github.com/scientific-python/specs/pull/323)
+- [SPEC-10: Changelog and release documentation](https://github.com/scientific-python/specs/pull/321)
+- [SPEC-12: Formatting mathematical expressions](https://github.com/scientific-python/specs/pull/326)
+- [SPEC-13: Naming conventions](https://github.com/scientific-python/specs/pull/324)
+
+Some existing SPECs were discussed and improved:
+
+- [SPEC-7: Seeding pseudo-random number generation (SPRaNG)](https://scientific-python.org/specs/spec-0007/)
+
+### Tooling
+
+- We created a new tools team to handle the ever-growing [list of tools we maintain](https://tools.scientific-python.org/).
+- Eric added his [circleci-artifacts-redirector-action](https://github.com/scientific-python/circleci-artifacts-redirector-action) to the suite.
+- Matthias brought over his [backport bot](https://github.com/scientific-python/MeeseeksDev) and set up a maintenance team.
+
+### SciPy
+
+Several of the SciPy developers were present, and we used the opportunity to celebrate Dan Schult joining as a core developer.
+Matt and Pamphile did some work on the new distribution infrastructure, Dan worked on sparse (remotely with CJ), and newly-supported [`const` statements were added to Cython code](https://github.com/scipy/scipy/pull/20891) got reviewed and merged.
+
+### Unplanned collaborations
+
+As is the nature of these events, some collaborations arise spontaneously.
+E.g., Nick and Ariel explored using Awkward Array for neuro-tractography, while Nick and Mridul explored using [scipp](https://scipp.github.io/index.html) for high-energy physics data.
+Guen worked on telemetry, while Inessa and Sanket discussed community surveys and governance.
+Sebastian and Thomas [discussed parallelization APIs](https://hackmd.io/84thx0ucQ2ab17ZYrBhWRw).
```

Comment:
`e.g.` is odd at the beginning of a sentence to me. Are all of these lines examples of the collaborations? In which case, should it be a bullet list?
```suggestion
As is the nature of these events, some collaborations arise spontaneously. For example,
- Nick and Ariel explored using Awkward Array for neuro-tractography, while Nick and Mridul explored using [scipp](https://scipp.github.io/index.html) for high-energy physics data.
- Guen worked on telemetry, while Inessa and Sanket discussed community surveys and governance.
- Sebastian and Thomas [discussed parallelization APIs](https://hackmd.io/84thx0ucQ2ab17ZYrBhWRw).
```

Reply:
I think this is OK, but I'll let others weigh in; let's see.

Reply:
To me, it seems strange to have bullet lists with more than one item in each bullet.  And splitting the sentences further loses the conversational feel of the blog. I suppose we could spell out whatever “e.g.” stands for, but “that’s Greek to me” (or maybe Latin?).  :)  While “e.g” and “for example” are literally translations of each other, the abbreviation feature means that “e.g.”  seems to be used in the middle of sentences rather than the beginning. (We’re not supposed to start a sentence with an abbreviation.)

How about starting with “For example,” instead of “E.g.,”

Reply:
Controversial suggestion, I know, but I feel it so often in many repos that all of them should use the no-bad-latin CI job we have over at the turning way ;) https://github.com/numpy/numpy-tutorials/blob/main/content/mooreslaw-tutorial.md?plain=1#L480-L496

Reply:
Expanded.

Reply:
Because the list now got long, the listification happened anyway.